datageneration/gpt_data_generator.py

```diff
@@ -325,26 +333,26 @@ def generate_written_word_distance(self, metric: str, max_digits: int) -> tuple:
         :param max_digits: maximum number of digits allowed
         :return: numeric - new numeric value, written - corresponding number in written words
         """
-        digits = randint(1, max_digits-2)
-        low = np.power(10, digits - 1)
-        high = np.power(10, digits) - 1
-
-        numeric = randint(low, high) * 100
-        written = num2words(numeric) + " " + metric
-        if np.random.choice([True, False]):
-            written = written.replace(",", "")
-        numeric = str(numeric) + " " + metric
-
+        # print("===DISTANCE===")
+        # print(distance)
+        # todo: why do we generate again???? i remove it
```

Comment:
We need to generate it again here. The original distance can be any number, but the written numbers are limited to steps of 100 (so you could for example not have a written number one-hundred twenty-three, as that is not a step of 100). Better revert this!